static/recommendation/css/recommendation.css

```diff
@@ -0,0 +1,68 @@
+html, body {
+    height: 100%;
+}
+
+body {
+    display: -ms-flexbox;
+    display: flex;
+    -ms-flex-align: center;
+    align-items: center;
+    padding-top: 40px;
```

Comment:
This might cause the UI to break (extra padding for recommendation tab). We can get rid of this statement.

Reply:
removed